src/Build.UnitTests/BackEnd/MSBuild_Tests.cs

```diff
@@ -374,10 +374,11 @@ public void PropertyOverridesContainSemicolon()
             ObjectModelHelpers.CreateFileInTempProjectDirectory(
                 @"bug'533'369\Sub;Dir\ConsoleApplication1\ConsoleApplication1.csproj", @"
 
-                <Project DefaultTargets=`Build` ToolsVersion=`msbuilddefaulttoolsversion` xmlns=`msbuildnamespace`>
```

Comment:
This removal is harmless and probably good but not directly related, to be clear.